packages/services/api/src/modules/audit-logs/providers/audit-logs-types.ts

```diff
@@ -327,6 +327,18 @@ export const AuditLogModel = z.union([
       scriptContents: z.string(),
     }),
   }),
+  z.object({
+    eventType: z.literal('ORGANIZATION_ACCESS_TOKEN_CREATED'),
+    metadata: z.object({
+      organizationAccessTokenId: z.string().uuid(),
+    }),
```

Comment:
@dotansimha Do you see it necessary to store more data here
e.g. 
- assigned permissions
- specified resources

Reply:
https://github.com/graphql-hive/console/pull/6493/commits/8c38327a078b12d24c6a73e91478a982d2c575a2